manifests/cf-manifest/manifest/041-security-groups.yml

```diff
@@ -1,30 +1,38 @@
-properties:
-  cc:
-    security_group_definitions:
-    - name: public_networks
-      rules:
-      - protocol: all
-        destination: 0.0.0.0-9.255.255.255
-      - protocol: all
-        destination: 11.0.0.0-169.253.255.255
-      - protocol: all
-        destination: 169.255.0.0-172.15.255.255
-      - protocol: all
-        destination: 172.32.0.0-192.167.255.255
-      - protocol: all
-        destination: 192.169.0.0-255.255.255.255
-    - name: dns
-      rules:
-      - protocol: tcp
-        destination: 0.0.0.0/0
-        ports: '53'
-      - protocol: udp
-        destination: 0.0.0.0/0
-        ports: '53'
+instance_groups:
+  - name: api
+    jobs:
+      - name: cloud_controller_ng
+        properties:
+          cc:
+            security_group_definitions:
+            - name: public_networks
+              rules:
+              - protocol: all
+                destination: 0.0.0.0-9.255.255.255
+              - protocol: all
+                destination: 11.0.0.0-169.253.255.255
+              - protocol: all
+                destination: 169.255.0.0-172.15.255.255
+              - protocol: all
+                destination: 172.32.0.0-192.167.255.255
+              - protocol: all
+                destination: 192.169.0.0-255.255.255.255
+            - name: dns
+              rules:
+              - protocol: tcp
+                destination: 0.0.0.0/0
+                ports: '53'
+              - protocol: udp
+                destination: 0.0.0.0/0
+                ports: '53'
 
-    default_running_security_groups: ["public_networks", "dns"]
-    default_staging_security_groups: ["public_networks", "dns"]
+            default_running_security_groups: ["public_networks", "dns"]
+            default_staging_security_groups: ["public_networks", "dns"]
 
-  garden:
-    deny_networks:
-      - 0.0.0.0/0
+  - name: diego-cell
+    jobs:
+      - name: garden
+        properties:
+          garden:
+            deny_networks:
+              - 0.0.0.0/0
```

Comment:
Does it make sense to move everything from this file into `010-cf-jobs.yml`? (I discovered this while following your instructions to compare the resulting differences with upstream).

Reply:
I did the minimum changes possible. But yeah, we could movi it.